src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingleadingasterisk/InputMissingLeadingAsterisk.java

```diff
@@ -0,0 +1,20 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingleadingasterisk;
+
+public class InputMissingLeadingAsterisk {
+  /**
+   ** OK
+   *
+       violation
+   violation
+
+   **/
```

Comment:
please add examples with tag and param on line start

Reply:
Done